Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -595,12 +595,12 @@ bool RenderLayerCompositor::updateCompositingPolicy()
     }
     
     static auto cachedMemoryPolicy = WTF::MemoryUsagePolicy::Unrestricted;
-    static MonotonicTime cachedMemoryPolicyTime;
-    static constexpr auto memoryPolicyCachingDuration = 2_s;
-    auto now = MonotonicTime::now();
-    if (now - cachedMemoryPolicyTime > memoryPolicyCachingDuration) {
+    static auto cachedMemoryPressureStatus = false;
```

Comment:
I'm not sure initializing to false here, and the `^=` below is correct; what if it starts high? I think you should read `cachedMemoryPressureStatus` each time. Also, a bool called `status` is confusing; not sure what this represents, exactly.

Reply:
How about this?

```C++
static bool cachedIsUnderMemoryPressureOrWarning = MemoryPressureHandler::singleton().isUnderMemoryPressure() || MemoryPressureHandler::singleton().isUnderMemoryWarning();
```

Then I think the `^=` should be correct after that.